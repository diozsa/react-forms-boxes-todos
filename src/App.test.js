import { render, screen } from '@testing-library/react';
import App from './App';

//smoke test
test('renders App without crashing', () => {
  render(<App />); 
});

//snapshot test
test('matches snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})