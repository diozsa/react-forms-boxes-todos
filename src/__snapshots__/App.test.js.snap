// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <ul>
        <div>
          <form>
            <label
              for="task"
            >
              New Task
            </label>
            <input
              id="task"
              name="task"
              placeholder="task"
              type="text"
              value=""
            />
            <button>
              Add Task!
            </button>
          </form>
        </div>
      </ul>
    </div>
    ;
  </div>
</DocumentFragment>
`;
